### PR TITLE
LPS-118140 Replace tabs with display style icon for publication review page

### PR DESCRIPTION
**Pull request by stsquared99**

### CODE REVIEW

File: modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/change_lists/js/ChangeTrackingChangesView.js [hunk 1]
```diff
@@ -614,6 +610,64 @@ class ChangeTrackingChangesView extends React.Component {
...
+		const context = {
+			label: Liferay.Language.get('context'),
+			onClick: () =>
+				this._handleNavigationUpdate({
+					nodeId: 0,
+					viewType: 'context',
+				}),
+			symbolLeft: 'pages-tree',
+		};
+
```
Comment (Preston-Crary) [inline modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/change_lists/js/ChangeTrackingChangesView.js:638]: Is this if needed?

**Comment by Preston-Crary** [comment]
When clicking on the dropdown and then clicking off it stays partly highlighted.

highlight

**Comment by Preston-Crary** [comment]
Please see those 2 things, if the highlighting issue takes too much time we can leave it for later.

File: modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/change_lists/js/ChangeTrackingChangesView.js (hunk 1 above)
Comment (stsquared99) [reply modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/change_lists/js/ChangeTrackingChangesView.js:638]: I was getting confused trying to get the active logic to behave as expected. Cleaned up a simplified now, thanks for catching that.

**Comment by stsquared99** [comment]
Resent https://github.com/Preston-Crary/liferay-portal/pull/1129

**Pull request closed without merging**